Doc/library/typing.rst

```diff
@@ -1033,6 +1107,26 @@ The module defines the following classes, functions and decorators:
    Note that returning instances of private classes is not recommended.
    It is usually preferable to make such classes public.
 
+.. decorator:: runtime_checkable
+
+   Mark a protocol class as a runtime protocol.
+
+   Such protocol can be used with :func:`isinstance` and :func:`issubclass`.
+   Raise :exc:`TypeError` if applied to a non-protocol class.  This allows
+   a simple-minded structural check very similar to one trick ponies in
```

Comment:
comma after "check", and put quotes around "one trick ponies" (it's a rather colloquial expression).